source/switching_to_live/index.html.md.erb

```diff
@@ -73,7 +70,7 @@ How you set up 3DS2 or 3DS1 payment authentication depends on which Payment Serv
 
 Choose which card types to accept by [selecting the account in the GOV.UK admin tool](https://selfservice.payments.service.gov.uk/my-services) and selecting **Settings**.
 
-If you use Worldpay, SmartPay or ePDQ, you must also make sure the same card types are selected in your PSP account.
+If you use Worldpay or ePDQ, you must also make sure the same card types are selected in your PSP account.
```

Comment:
Is there a way to avoid the passive voice here? 'You must also make sure you select/have selected the same card types...'?